.github/workflows/release.yml

```diff
@@ -0,0 +1,42 @@
+name: Release
+
+on:
+  release:
+    types: [published]
+
+jobs:
+  create-release:
```

Comment:
I'm a bit perplexed. This job seems to be creating a release. Do we need it if we're already made a release to trigger this workflow?

Reply:
It turns out this section is redundant. Even when it's here in the file, it won't actually create a new release (though it shows up as an action in the workflow). I've removed it, thanks.

Reply:
Also, there are no restrictions on the format of the tag with this change.